File: Python/1115.py
nPupils, nApples = map(int, input().split())

appleByPupil = nApples // nPupils
appleInBasket = nApples % nPupils
resentPupil = (nPupils - appleInBasket) % nPupils

print(appleByPupil, appleInBasket, resentPupil)

File: C++/1115.cpp
#include <iostream>
using namespace std;


int main()
{
    int nPupils, nApples, appleByPupil, appleInBasket, resentPupil;

    cin >> nPupils >> nApples;

    appleByPupil = nApples / nPupils;
    appleInBasket = nApples % nPupils;
    resentPupil = (nPupils - appleInBasket) % nPupils;

    cout << appleByPupil << " " << appleInBasket << " "
        << resentPupil << endl;

    return 0;
}
